pkg/config/config.go

```diff
@@ -65,6 +68,22 @@ func (w *IniWriter) Write(section string, keyValue map[string]string) error {
 		w.File.Section(section).Key(key).SetValue(value)
 	}
 
+	releaser, err := mutex.Acquire(mutex.Spec{
+		Name:    "wakatime-cli-config-mutex",
+		Delay:   time.Millisecond,
+		Timeout: defaultTimeout,
+		Clock:   &mutexClock{delay: time.Millisecond},
+	})
+	if err != nil {
+		log.Debugf("failed to acquire mutex: %s", err)
```

Comment:
This is perfect, in case something goes wrong we should still try and continue writing the configs and saving heartbeats to offline db 👍